devops/proxies-remove

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+yarn config delete proxy
```

Comment:
echo "Removing http_proxy for Yarn..."
...echo "Removing https_proxy for Yarn..."

Reply:
O comando do yarn já imprime inforamções no terminal:
````
Bine@Bine-PC MINGW64 ~
$ proxies-apply
Bine@Bine-PC MINGW64 ~
$ proxies-apply
yarn config v1.5.1
success Set "proxy" to "http://proxy:8080".
Done in 0.05s.
yarn config v1.5.1
success Set "https-proxy" to "http://proxy:8080".
Done in 0.05s.
Applying http_proxy for GIT...
Applying https_proxy for GIT...
Applying http_proxy for Linux terminal...
Applying https_proxy for Linux terminal...
All done, proxies applied.
````

Reply:
Certo. 👍